module/enrichers.mjs

```diff
@@ -25,6 +25,12 @@ export function registerEnrichers() {
       pattern: /\[\[\/language (\w+)]]/g,
       enricher: enrichLanguage
     },
+    {
+      id: "dnd5eAward",
+      pattern: /\[\[\/award ([\w\s]+)]]/g,
```

Comment:
I think the default behavior needs to be that the award is split (divided) among the party members and that we need to support an `each` keyword that is used to designate cases where every recipient receives the award.

For milestones awards, of course, `each` behavior is default.